src/Storage/Storage/Common/Cmdlet/NewAzureStorageContext.cs

```diff
@@ -310,6 +324,12 @@ internal CloudStorageAccount GetAnonymousStorageAccount(string storageAccountNam
             return GetStorageAccountWithEndPoint(credential, storageAccountName, useHttps, storageEndpoint);
         }
 
+        internal CloudStorageAccount GetStorageAccountWithCustomEndpoint(bool useHttps, string customEndpoint)
+        {
+            StorageCredentials credential = new StorageCredentials();
```

Comment:
This method was developed similarly to the method GetAnonymousStorageAccount() as it doesn't require a credential to connect to the storage account.

Reply:
The change only support customer endpoint with Anonymous credentail.
We should also support it for key/SAS/Oauth credentail.